changelog/db.changelog-1.1.1.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.1" encoding="utf-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Marcin Mogiela" id="1">
+    <tagDatabase tag="v1.1.1" />
```

Comment:
Is there a way we can only apply this to the local environment possibly through Docker compose? I think we will only want this dataset for development purposes and possible don't want it as a liquidbase migration unless we can limit where it runs

Reply:
I think we can use liquibase context here

<img width="889" alt="image" src="https://user-images.githubusercontent.com/118533775/206128336-5e279da6-8a45-44f4-8c4b-d5b3376e0f14.png">

Reply:
<img width="682" alt="image" src="https://user-images.githubusercontent.com/118533775/206128959-ae9a7783-655c-40d5-8587-844e783a8609.png">

Reply:
<img width="596" alt="image" src="https://user-images.githubusercontent.com/118533775/206130139-8f74d9ef-b514-4113-8d59-dee940ef2399.png">